src/main/java/stringcalculator/domain/Delimiter.java

```diff
@@ -0,0 +1,47 @@
+package stringcalculator.domain;
+
+import static stringcalculator.domain.Validator.validatePositiveNumber;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Delimiter {
+
+    private final static String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private final static String[] DEFAULT_DELIMITER = new String[]{",", ":"};
+
+    private static final List<String> delimiters = new ArrayList<>(
+        Arrays.asList(DEFAULT_DELIMITER));
```

Comment:
`addCustomDelimiter`를 할때 `unsupoortOperationException`이 발생해서 new ArrayList로 한번 더 wrapping하신게 맞을까요? 
현재 로직은 List를 두번 생성하게 되는 문제가 있는 것 같은데요~!!
이 경우에는 Stream을 사용하여 변환을 해보면 어떨까요?? [참고](https://www.baeldung.com/java-8-collectors)

```suggestion
    private static final List<String> delimiters =
        Arrays.stream(DEFAULT_DELIMITER).collect(Collectors.toList());
```

Reply:
👍